lib/generators/provider/templates/.travis.yml

```diff
@@ -9,6 +9,7 @@ env:
   - RUBY_GC_HEAP_GROWTH_MAX_SLOTS=300000
   - RUBY_GC_HEAP_INIT_SLOTS=600000
   - RUBY_GC_HEAP_GROWTH_FACTOR=1.25
+bundler_args: --no-deployment
```

Comment:
https://github.com/ManageIQ/manageiq/pull/15508/files#diff-7dbc55ccfbebe14a0aa870a17989c558R29 is the first bundle that would occur in bin/setup.  Then, you get a second bundle update as the default "install" step on travis.  Specifically, under the covers it does `bundle install --jobs=3 --retry=3 --deployment`, and the reason you get the --deployment bit is because the Gemfile.lock exists after the first bundle install.  See https://docs.travis-ci.com/user/languages/ruby/#Travis-CI-uses-Bundler and https://docs.travis-ci.com/user/customizing-the-build/#Skipping-the-Installation-Step